[2022-01-25 10:29:44,866] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T10:29:28.085292+00:00 [queued]>
[2022-01-25 10:29:45,132] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T10:29:28.085292+00:00 [queued]>
[2022-01-25 10:29:45,155] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 10:29:45,175] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 10:29:45,193] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 10:29:45,270] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): load_data> on 2022-01-25T10:29:28.085292+00:00
[2022-01-25 10:29:45,343] {standard_task_runner.py:52} INFO - Started process 17822 to run task
[2022-01-25 10:29:45,377] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'load_data', '2022-01-25T10:29:28.085292+00:00', '--job-id', '74', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpopevcomo', '--error-file', '/tmp/tmpacixy_1n']
[2022-01-25 10:29:45,389] {standard_task_runner.py:77} INFO - Job 74: Subtask load_data
[2022-01-25 10:29:45,581] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.load_data 2022-01-25T10:29:28.085292+00:00 [running]> on host 2dc98551bc76
[2022-01-25 10:29:45,734] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T10:29:28.085292+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T10:29:28.085292+00:00
[2022-01-25 10:30:13,708] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 10:30:13,728] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=load_data, execution_date=20220125T102928, start_date=20220125T102944, end_date=20220125T103013
[2022-01-25 10:30:13,783] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 10:30:13,826] {local_task_job.py:146} INFO - Task exited with return code 0
